pkg/util/bs/bs.go

```diff
@@ -0,0 +1,19 @@
+package bs
```

Comment:
Nit: Let's not use `bs` as a package name. That doesn't provide any useful indication of purpose and can have "bad" connotations. 

Maybe: `convert` or `zerocopy`, so that the calls become `convert.StringToBytes` or `zerocopy.StringToBytes` ?

Reply:
bbe1d75